src/telemetry/os_info.rs

```diff
@@ -100,4 +100,46 @@ BUG_REPORT_URL="https://bugs.debian.org/""#;
         assert_eq!(data["/osName"], "Debian GNU/Linux");
         assert_eq!(data["/osVersion"], "11");
     }
+
+    #[test]
+    fn os_release_with_only_name() {
+        let file = r#"NAME="Arch Linux"#;
+
+        let data = parse_os_info(file).unwrap();
+        assert_eq!(data["/osName"], "Arch Linux");
+        assert!(!data.contains_key("/osVersion"));
+    }
+
+    #[test]
+    fn os_release_malformed() {
+        let file = r#"NAM["Arch Linux"@@"#;
+
+        let data = parse_os_info(file).unwrap();
+        assert!(data.is_empty());
+    }
+
+    #[test]
+    fn parse_key_value_line_empty() {
+        let line = "";
+
+        let data = parse_key_value_line(line);
+        assert!(data.is_none());
+    }
+
+    #[test]
+    fn parse_key_value_line_bad() {
+        let line = r#"OS;"Arch"#;
+
+        let data = parse_key_value_line(line);
+        assert!(data.is_none());
+    }
+
+    #[test]
+    fn parse_key_value_line_good() {
+        let line = r#"OS="Arch"#;
+
+        let (key, value) = parse_key_value_line(line).unwrap();
+        assert_eq!(key, "OS");
+        assert_eq!(value, "Arch");
+    }
```

Comment:
Previous tests are named `os_release_with_only_name` and `os_release_malformed`. Could these two be called `parse_key_value_line_malformed` and `parse_key_value_line` for consistency?